snap/local/files/run_nv_hostengine.sh

```diff
@@ -12,3 +12,9 @@ if [ -n "$nv_hostengine_port" ]; then
 fi
 
 exec "$SNAP/usr/bin/nv-hostengine" -n --service-account nvidia-dcgm "${args[@]}"
+
```

Comment:
iiuc the lines here will never execute, because the exec in the line above will replace the process running this script

Reply:
But it does...

Reply:
Maybe daemon on-fail fires up

Reply:
interesting... perhaps worth testing with a print. I may be wrong, I thought there's no going back from an exec

Reply:
Yeah these lines shouldn't execute; if they do, there's something really wrong with `exec` here :thinking: :thinking:

Reply:
Yeah, it was a fluke on my side that made me sure that everything works :)